src/assets/translations/en-us.yaml

```diff
@@ -261,6 +261,17 @@ containerRuntime:
       label: containerd
       description: Namespaces for container images; use with nerdctl.
 
+pathManagement:
+  label: Configure PATH
+  tooltip: Choose how you'd like Rancher Desktop to look for Supporting Utilities.
+  options:
+    rcFiles:
+      label: Automatic
```

Comment:
I think we landed on `Automatic` and `Manual` since we only have to two options. I'm open to adding `(easiest)`, but I think the challenge level is expressed adequately through the existing option.

Reply:
Sure, I'm good with it either way.